### ISSUE TITLE
Specifying mode removes every variable

### ISSUE THREAD

**Issue by padcom** [lxs24sxl/vite-plugin-html-env#3]
in Vite the `mode` can be specified in two ways:

```
npm run build -- --mode production
```
or
```
npm run build -- --mode=production
```
When using the latter syntax, no values are present in the resulting `index.html`.

**Comment by lxs24sxl** [comment]
Thank you for the question, I have fixed this issue in 1.0.3.

Regarding the question why not get the mode parameter from defineConfig, I just want to make it easier for users.

**Comment by padcom** [comment]
Thank you! You're awesome!
